sdk/keyvault/keyvault-common/src/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export * from "./challengeBasedAuthenticationPolicy";
```

Comment:
can you confirm that the policy correctly is showing up via api-extractor's review.md for certificates with this?

Reply:
This policy is not part of the public API of the client. When I build certificates, there are no unstaged changes, so I believe this is good! Let me know if I'm missing something.

Reply:
oh gotcha